wear/src/main/java/io/homeassistant/companion/android/home/MainViewModel.kt

```diff
@@ -31,6 +33,7 @@ import javax.inject.Inject
 @HiltViewModel
 class MainViewModel @Inject constructor(
     private val favoritesDao: FavoritesDao,
+    private val FavoriteCachesDao: FavoriteCachesDao,
```

Comment:
Variable names usually start with a lowercase letter :)

```suggestion
    private val favoriteCachesDao: FavoriteCachesDao,
```